src/Symfony/Component/Form/Extension/Core/DataTransformer/WeekToArrayTransformer.php

```diff
@@ -33,7 +33,7 @@ class WeekToArrayTransformer implements DataTransformerInterface
      */
     public function transform($value)
     {
-        if (null === $value) {
+        if (null === $value || '' === $value) {
```

Comment:
Can you debug how `$value` can be the empty string here instead of `null` (adding a failing test case could also be an option)?

Reply:
Actually, there is a failing test now.